### ISSUE TITLE
scp does not overwrite files

### ISSUE THREAD

**Issue by ronsmits** [hierynomus/sshj#311]
Hi

I am using your project in a maven plugin (https://github.com/ronsmits/fxlauncher-maven-plugin) it seems that `sshClient.newSCPFileTransfer().upload(new FileSystemFile(buildDir), path);` does not overwrite existing files. Is it somehow possible to do this? am I missing a setting?

**Comment by hierynomus** [comment]
Hi @ronsmits,

What is the exception you're getting? I've not seen this, and also cannot remember having this problem. It might be ssh server dependent? Is the target system Windows?

**Comment by ronsmits** [comment]
There is no exception, the files are just not written. The server is a raspberry pi running raspbian. I already checked if I could find something int scpCommandLine that could be set, but there is not. 

Information on the interweb is a bit sketchy wether the standard behaviour is to overwrite or silently ignore, Most seem to go towards the overwrite.

**Comment by hierynomus** [comment]
Yes indeed, from what I know overwrite is the standard behaviour, except in
some corner cases. Do you have a small standalone reproduction case, it
might be how you specify the paths (with or without trailing slash)

2017-03-27 14:14 GMT+02:00 Ron Smits <notifications@github.com>:

**Comment by bendem** [comment]
It would be great if the solution implemented to solve this problem allowed both behaviours.
